style.css

```diff
@@ -515,3 +942,472 @@ body {
+    width: 38px;
+    height: 38px;
+    -webkit-transform: translate(-4px, -9%);
+    -ms-transform: translate(-4px, -9%);
+    transform: translate(-4px, -9%);
+  }
+
+  .middle-section__arrow {
+    cursor: pointer;
+    position: relative;
+    left: 240px;
+    bottom: 25px;
+  }
+
+  .middle-section__arrow div {
+    position: relative;
+    top: 18px;
+    width: 240px;
+    height: 2px;
+    background-color: #000;
+    display: block;
+  }
+
+  .middle-section__arrow div::after {
+    content: '';
+    position: absolute;
+    width: 7px;
+    height: 1px;
+    top: -3px;
+    right: 0;
+    background-color: #000;
+    transform: rotate(45deg);
+  }
+
+  .middle-section__arrow div::before {
+    content: '';
+    position: absolute;
+    width: 7px;
+    height: 1px;
+    top: 4px;
+    right: 0;
+    background-color: #000;
+    box-shadow: 0 3px 5px rgba(0, 0, 0, .2);
+    transform: rotate(-45deg);
+  }
+
+  .quantity,
+  .middle-section__btn_other {
+    display: block;
+    color: #000;
+  }
+
+  .quantity {
+    width: 100px;
+    height: 44px;
+    padding-top: 12px;
+    margin-right: 15px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    border: 1.5px solid #000000;
+  }
+
+  .middle-section__minus,
+  .middle-section__plus {
+    border: none;
+    cursor: pointer;
+  }
+
+  .middle-section__minus:hover {
+    cursor: pointer;
+    background-color: #c29585;
+  }
+
+  .middle-section__plus:hover {
+    background-color: #c29585;
+    cursor: pointer;
+  }
+
+  .middle-section__value {
+    margin: 0 10px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+  }
+
+  .form__colors-item:hover {
+    transform: scale(1.2);
+  }
+
+  .middle-section__subtitle {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 16px;
+    letter-spacing: 0.07em;
+  }
+
+  .middle-section__price{
+    margin-right: 22px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 56px;
+  }
+
+  .middle-section__title {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 41px;
+  }
+
+  .middle-section__desription {
+    line-height: 21px;
+    font-size: 15px;
+    font-weight: normal;
+    opacity: 0.5;
+  }
+}
+
+@media (min-width: 768px) {
+  .middle-section {
+    margin: 0;
+    position: relative;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    padding-top: 114px;
+    padding-left: 40px;
+    padding-right: 40px;
+  }
+
+  .middle-section__image {
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    -webkit-box-flex: 2;
+    -ms-flex: 2;
+    flex: 2;
+    height: 100%;
+    margin: 0;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)